pages/index.tsx

```diff
@@ -98,6 +99,18 @@ const Icebreaker: NextPage<Props> = ({
           handleCopyUrlClick={handleCopyUrlClick}
         />
       </Card>
+      <div
+        className="group mx-4 mt-auto mb-4 inline-flex flex-col items-center justify-center gap-2 text-center text-white"
+      >
+          <a
+            className="text-sm underline mt-4 inline-block"
```

Comment:
+1 let's add some hover state so we can get a little visual indication when you hover over the link - https://tailwindcss.com/docs/hover-focus-and-other-states